### PR TITLE
[ebaycommercenetwork.com] New ruleset

### PR DESCRIPTION
**Pull request by youdly**
https://github.com/EFForg/https-everywhere/pull/13773

### CODE REVIEW

**Comment by jeremyn** [comment]
- [ ] Add a `test` for https://productads.ebaycommercenetwork.com or remove the `target` and mark it as `No working URL known` in the top comment
- [ ] Change the `ruleset` `name` to `EBayCommerceNetwork.com`
- [ ] Change the filename to `EBayCommerceNetwork.com.xml`

File: src/chrome/content/rules/ebaycommercenetwork.com.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!--
+	For other eBay coverage, see ebay.com.xml
+
+
+	Non-functional subdomains:
+		- $self		(r)
```
Comment (J0WI) [inline src/chrome/content/rules/ebaycommercenetwork.com.xml]: same for `www`

Comment (jeremyn) [reply src/chrome/content/rules/ebaycommercenetwork.com.xml]: I agree and have now added "Mark https://www.ebaycommercenetwork.com as `Refused` in the top comment." to the checklist. Thanks.

**Comment by J0WI** [comment]
@youdly this is blocking the other #13603 PRs. Could you have a look here?

**Comment by jeremyn** [comment]
@youdly I'd like to finish this pull request. If we don't hear from you soon, I'm going to go ahead and make a new pull request myself for these domains, and close this pull request.

**Comment by jeremyn** [comment]
I'm closing this pull request in favor of my new pull request https://github.com/EFForg/https-everywhere/pull/14549.

**Pull request closed without merging**